notm-specs/src/test/resources/GebConfig.groovy

```diff
@@ -65,4 +65,4 @@ reportsDir = "build/geb-reports"
 reportOnTestFailureOnly=true
 
 // Close browser on shutdown - uncomment to enable
-// quitCachedDriverOnShutdown = false
+quitCachedDriverOnShutdown = false
```

Comment:
is this intentional?

Reply:
:man_facepalming: no, good spot, will fix